hatch/src/cli/commands/new.rs

```diff
@@ -63,6 +70,13 @@ impl<'command> Command<'command> for New {
            .help("Set the project version")
            .long("version").short("v").takes_value(true)
            .required(false))
+
+      .arg(Arg::with_name("include")
+           .help("List URLs to git repositories")
+           .long("include")
+           .short("i")
+           .min_values(1)
+           .takes_value(true))
```

Comment:
Can add a `.required(false)` to this to be consistent with the rest of the Args?
Also, can we structure this like the rest of the Args?  
```
.help("List URLs to git repositories")
.long("include").short("i").min_values(1).takes_value(true)
.required(false))
```